src/main/java/seedu/duke/utils/parsers/AddParser.java

```diff
@@ -40,12 +40,18 @@ public void run() {
             BigDecimal price = new BigDecimal(matcher.group(PRICE_INDEX));
             String category = matcher.group(CAT_INDEX);
 
-            if (quantity.compareTo(BigInteger.ONE) < 0 ||
+      /*      if (quantity.compareTo(BigInteger.ONE) < 0 ||
```

Comment:
If the code is redundant maybe can removed it already since we are refactoring out all legacy and commented out code

Reply:
removed commented code